src/bots/google/Gemini20ExpAPIBot.js

```
@@ -0,0 +1,11 @@
+import GeminiAPIBot from "./GeminiAPIBot";
+
+export default class Gemini20ExpAPIBot extends GeminiAPIBot {
+  static _className = "Gemini20ExpAPIBot";
+  static _logoFilename = "gemini-2.0-exp-logo.png";
+  static _model = "gemini-exp-1206";
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)